response.go

```diff
@@ -76,6 +130,10 @@ func (f FileResponse) GetContentType() string {
 	return resolveContentType(f.ContentType, contentTypeFromFileName(f.FilePath))
 }
 
+func (f FileResponse) GetContent() any {
+	return f.FileContent
```

Comment:
again FileContent should not exist. WriteContent is meant to read the file at runtime.